README.md

```diff
@@ -11,24 +11,204 @@ This doc will describe how to run your own private Tezos blockchain.
+      Secret Key: unencrypted:edsk3mXNLyaNXdFv6Qjcxmfed3eJ7kSzJwgCjSNh4KTTpwRRLPMSpY
+```
+### Activating the procotol and starting the blockchain
+The Public Key from the previous step will now need to be pasted into a JSON paramter file.
+
+Two sample JSON files are provided, depending on the version of the network you plan to run:
+* ./parameters/parameters_babylonnet.json
+* ./parameters/parameters_carthagenet.json
+
+In these files, `bootstrap_accounts` has information about account public keys
+that have access to tokens (4M of tez in these example files). Note
+that all bakers should have some tokens, thus, we need to add the public key for the baker just created into `bootstrap_accounts`.
 
-### Running scripts outside docker prerequisites
+Starting with the appropriate sample file for the network version you plan to run,
+modify it by adding an entry in the bootstrap_accounts section for the public key provided in the previous step.
+
+e.g. paste an entry like this into the bootstrap_accounts section:
+```sh
+    [
+      "edpkvRTXYRCxCbWs4GF1shMxCab9nF3iNimPqqb2esiP5WyjAhT1dz",
+      "4000000000000"
+    ],
+```
+The exisiting bootstrap accounts should remain in the file, and will be used later in this example.
+
+Copy the edited parameters file to the docker filesystem:
+```sh
+docker cp my-parameters.json <container_name>:/parameters.json
+```
+where my-parameters.json is the file you have just edited. The <container_name> can be retrieved by the command 'docker ps'
+
+The last step is to run the activation script for the running docker container:
+```sh
+docker exec <container_name> ./scripts/activate-protocol.sh \
+  --base-dir /base-dir --tezos-client /base-dir/tezos-client \
+  --parameters /parameters.json
+```
 
+If you want to browse the file system inside your Docker container, you can run the command:
+```sh
+docker exec -it <container_name> bash
+```
+This will run an interactive shell session inside your running node container.
+
+### Using the private chain
+Before continuing, you can verify things are working properly by entering into your browser:
+```sh
+http://localhost:8732/chains/main/blocks/head
+```
+You should see some valid JSON being returned. If things are not working correctly, you can look at the contents of the files
+```sh
+base-dir/baker.log
+base-dir/node.log
+```
+for possible error messages.
+
+
+Once the protocol is activated, you can play with the new chain.
+For example, you can transfer some tokens from one account to another using `tezos-client`.
+
+We will use the alias 'alice' to refer to the bootstrap_accounts entry with these values:
+```sh
+Hash: tz1akcPmG1Kyz2jXpS4RvVJ8uWr7tsiT9i6A
+Public Key: edpktezaD1wnUa5pT2pvj1JGHNey18WGhPc9fk9bbppD33KNQ2vH8R
+Secret Key: unencrypted:edsk2vKVH2BNwKrxJrvbRvuHnu4FW17Jrs2Uy2TzR2fxipikTJJ1aG
+```
+Enter the following command:
+```sh
+$ tezos-client import secret key alice unencrypted:edsk2vKVH2BNwKrxJrvbRvuHnu4FW17Jrs2Uy2TzR2fxipikTJJ1aG
```

Comment:
I think it worth saying that one can use `tezos-client` which is located in the container or obtain one from https://github.com/serokell/tezos-packaging

Reply:
Ok, done.